docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,53 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
+  Reference](./saml-reference.mdx) before proceeding.
+- User with permission to create Service Provider resource. The preset `editor` role has this permission.
+- Test SAML application, also known as SAML Service Provider (SP). For this guide, we'll be using
```

Comment:
What does "also known as SAML Service Provider (SP)" mean here? Does this rename "Test SAML application"?

Reply:
Not really, but I updated this line so to avoid such confusion:
`SAML application (also known as a SAML service provider or SP) for testing. For this guide, we'll be using
[samltest.id](https://samltest.id/) as a service provider for Teleport's SAML IdP.`